pages/en/lb4/Getting-started.md

```diff
@@ -46,12 +54,67 @@ const app = new Application({
   await app.start();
   console.log(`REST server listening on port ${server.getSync('rest.port')}`);
 })();
+
+```
+
+Next, create a file called tsconfig.json to indicate that this is a TypeScript project and specify the TypeScript compiler options required.  For more details, see [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
+
+{% include code-caption.html content="tsconfig.json" %}
+```
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "es2017",                          
+    "module": "commonjs",                     
+
+    /* Experimental Options */
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true
+  }
+}
+```
+
+Finally, create a file called package.json. 
+
+{% include code-caption.html content="package.json" %}
+```
+{
+  "name": "getting-started",
+  "description": "Getting Started for LoopBack 4",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "tsc && node index.js"
+  },
+  "dependencies": {
+    "@loopback/core": "^4.0.0-alpha.16",
+    "@loopback/rest": "^4.0.0-alpha.3"
+  },
+  "devDependencies": {
+    "@types/node": "^8.0.34",
+    "typescript": "^2.5.3"
+  }
+}
+```
+
+### Running the application
+
+To install the required packages, enter:
+```
+npm install
 ```
 
-The example above creates an `Application` and a `RestServer` that responds to all HTTP requests with the text "Hello World".
+To run the application, enter:
+```
+npm start
+```
+
+On another terminal, enter:
+```
+curl http://localhost:3000/helloworld
+```
```

Comment:
Is there an advantage to using `curl` vs. just loading `http://localhost:3000/helloworld` in a browser?
Perhaps just add:
```
"... or load http://localhost:3000/helloworld in your web browser."